### ISSUE TITLE
CLI version flag

### ISSUE THREAD

**Issue by SeanHansell** [Lord-Kamina/SwiftDefaultApps#77]
`swda -v` returns: ```Version: 1.0``` even on v2.0.1. I suppose it's possible the CLI hasn't changed since the initial release but I believe its still best practice to increment version on all components to correspond with the understood "primary" version.

**Comment by Lord-Kamina** [comment]
Yeah, that was most likely an omission. I'll get around to fixing it eventually or, if you want, make a PR (Ideally, using the actual version string from the Info.plist, instead of hardcoding it)
